web/pdf_viewer.js

```diff
@@ -295,7 +295,6 @@ var PDFViewer = (function pdfViewer() {
       var onePageRendered = new Promise(function (resolve) {
         resolveOnePageRendered = resolve;
       });
-      this.onePageRendered = onePageRendered;
```

Comment:
Alright, let's keep this one then. I have updated the commit to put this line back. Note that the test run should still be fine because only this file changed (in `/web`, outside of the tests).